apps/laboratory/src/pages/library/ethers-all-internal.tsx

```diff
@@ -0,0 +1,42 @@
+import { createWeb3Modal, defaultConfig } from '@web3modal/ethers/react'
+import { ThemeStore } from '../../utils/StoreUtil'
+import { EthersConstants } from '../../utils/EthersConstants'
+import { ConstantsUtil } from '../../utils/ConstantsUtil'
+import { EthersTests } from '../../components/Ethers/EthersTests'
+import { Web3ModalButtons } from '../../components/Web3ModalButtons'
+import { siweConfig } from '../../utils/SiweUtils'
+import { SiweData } from '../../components/Siwe/SiweData'
+import { EthersModalInfo } from '../../components/Ethers/EthersModalInfo'
+
+const modal = createWeb3Modal({
+  ethersConfig: defaultConfig({
+    metadata: ConstantsUtil.Metadata,
+    defaultChainId: 1,
+    rpcUrl: 'https://cloudflare-eth.com',
+    auth: {
+      socials: ['google', 'x', 'discord', 'apple', 'github']
+    },
+    coinbasePreference: 'smartWalletOnly'
+  }),
+  chains: EthersConstants.chains,
+  projectId: ConstantsUtil.ProjectId,
+  enableAnalytics: true,
+  metadata: ConstantsUtil.Metadata,
+  termsConditionsUrl: 'https://walletconnect.com/terms',
+  privacyPolicyUrl: 'https://walletconnect.com/privacy',
+  siweConfig,
+  customWallets: ConstantsUtil.CustomWallets
```

Comment:
Should we add all other possible features?
- `tokens`
- `customWallets`
- `allWallets`
- `includeWalletIds`
- `excludeWalletIds`
- `featuredWalletIds`
- `allowUnsupportedChain`

Reply:
Same for wagmi

Reply:
Not sure if we need to add those, as these are more specific configuration adjustments than actual features. But I agree that we should test these somewhere.